src/master-playlist-controller.js

```diff
@@ -793,6 +798,46 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
+   * Check if a playlist has stopped being updated
+   * @param {Object} playlist the media playlist object
+   * @return {boolean} whether the playlist has stopped being updated or not
+   */
+  stopUpdateCheck(playlist) {
```

Comment:
minor, but I think we can come up with a better name for the method. Maybe `isPlaylistOutdated_`, `isPlaylistStagnant_`, `isPlaylistUpdating` or something along these lines that is a bit more descriptive of what the method does. (note the return value should reflect the name logically). This method should also be private, indicated by the `_` appended to the end of the name